pkg/statistics/handle/initstats/load_stats.go

```diff
@@ -49,7 +49,7 @@ func NewWorker(
 
 // LoadStats loads stats concurrently when to init stats
 func (ls *Worker) LoadStats(is infoschema.InfoSchema, cache statstypes.StatsCache, rc sqlexec.RecordSet) {
-	concurrency := runtime.GOMAXPROCS(0)
+	concurrency := max(2, runtime.GOMAXPROCS(0)/2)
```

Comment:
We set an upper bound for the concurrency? TiDB may be set up in a node with 64c, especially for OP cases?

Reply:
I add max ```16``` concurrency limit.